draft-ietf-teep-architecture.md

```diff
@@ -1134,21 +1123,22 @@ for protecting the resource usage allocated for TA management.
 
 ## Compromised CA
 
-A root CA for TAM certificates might get compromised. A Trust Anchor other 
-than a root CA certificate may also be compromised. Some TEE Trust
+TAMs are responsible for validating the TEE certificate chain, including
+the TEE certificate and any intermediate CA certificates up to the root
+certificate. Such validation includes checking for certificate revocation.
+A root CA for TAM certificates might get compromised or its certificate might
+expire. A Trust Anchor other 
+than a root CA certificate may also expire or be compromised. 
+If the Trust Anchor used by a TEE device certificate expires or is revoked, the
```

Comment:
Continuing the TEE certificates validation, these may mean "trust anchors" stored by TAM for validating TEE certificates, right?

Reply:
Please check updated text

Reply:
The entire paragraph is still mixed between the root CA for TAM certificates and the root CA for TEE certificates. Let me propose a revision.